### PR TITLE
Refactor package

### PR DESCRIPTION
**Pull request by fernandohds564**

### CODE REVIEW

File: apsuite/__init__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 __all__ = (
-    'commissioning_scripts', 'loco', 'optics_analysis', 'optimization',
+    'commisslib', 'loco', 'optics_analysis', 'optimization',
```
Comment (xresende) [inline apsuite/__init__.py:3]: nice!

**Review by xresende — APPROVED** [review]

File: apsuite/orbcorr/orbit_correction.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    def get_kicks(self):
+        """Return corrector kicks."""
+        if model is None:
+            model = self.respm.model
+        chidx = self.respm.ch_idx
+        cvidx = self.respm.cv_idx
+        rfidx = self.respm.rf_idx
+        kickch = pyaccel.lattice.get_attribute(model, 'hkick_polynom', chidx)
+        kickcv = pyaccel.lattice.get_attribute(model, 'vkick_polynom', cvidx)
+        kickrf = pyaccel.lattice.get_attribute(model, 'rf_frequency', rfidx)
```
Comment (murilobalves) [inline apsuite/orbcorr/orbit_correction.py:54]: The attribute is `frequency` instead of `rf_frequency`

File: apsuite/orbcorr/orbit_correction.py [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+        ncv = len(self.respm.cv_idx)
+
+        kickch, kickcv = kicks[:nch], kicks[nch:ncv]
+
+        for i, idx in enumerate(self.respm.ch_idx):
+            model[idx].hkick_polynom = kickch[i]
+        for i, idx in enumerate(self.respm.cv_idx):
+            model[idx].vkick_polynom = kickcv[i]
+        if self.params.userf:
+            model[self.respm.rf_idx[0]].rf_frequency = kicks[-1]
```
Comment (murilobalves) [inline apsuite/orbcorr/orbit_correction.py:144]: Same here

**Review by murilobalves — CHANGES_REQUESTED** [review]

**Review by anacso17 — APPROVED** [review]

File: apsuite/orbcorr/orbit_correction.py (hunk 3 above)
Comment (fernandohds564) [reply apsuite/orbcorr/orbit_correction.py]: fixed!

File: apsuite/orbcorr/orbit_correction.py (hunk 2 above)
Comment (fernandohds564) [reply apsuite/orbcorr/orbit_correction.py]: fixed!

**Review by murilobalves — APPROVED** [review]

**Pull request merged**